community/protobuf/build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+./configure --prefix=/usr
+make
+
+make DESTDIR="$DESTDIR" install
+install -vDm 644 LICENSE            -t "$1/usr/share/licenses/protobuf"
+install -vDm 644 editors/proto.vim  -t "$1/usr/share/vim/vimfiles/syntax"
```

Comment:
The verbose (`-v`) flag is usually not used in build files

Reply:
updated